app/reducers/room.js

```diff
@@ -47,6 +48,13 @@ export default function (state = initialState, action) {
 				...state,
 				isDeleting: false
 			};
+		case ROOM.UPLOADING_SEND:
+			return { ...state, uploadingSend: { ...state.uploadingSend, [action.name]: action.intervalValue } };
+		case ROOM.UPLOADING_REMOVE:
+			const uploadingSendState = Object.assign({}, state.uploadingSend);
+			clearInterval(uploadingSendState[action.name]);
```

Comment:
Reducers should not perform side-effects https://redux.js.org/usage/structuring-reducers/prerequisite-concepts